spec/annotation_spec.rb

```diff
@@ -74,6 +74,43 @@
             end
           end
         end
+
+        describe 'one annotation with all the keywords' do
+          subject do
+            Annotation.new("@probedock(key=#{quote}k#{quote} category=#{quote}c#{quote} active=#{quote}f#{quote}
+              tag=#{quote}t1#{quote} tag=#{quote}t2#{quote} ticket=#{quote}ti1#{quote} ticket=#{quote}ti2#{quote})")
+          end
+
+          it 'should be possible' do
+            expect(subject.key).to eq('k')
+            expect(subject.category).to eq('c')
+            expect(subject.active).to be_falsey
+            expect(subject.tags).to match_array(%w[t1 t2])
+            expect(subject.tickets).to match_array(%w[ti1 ti2])
+          end
+        end
+
+        describe 'multiple annotations with all the keywords' do
+          subject do
+            Annotation.new("
```

Comment:
When I added this test, I encountered for tags and tickets (probably) when `#{quote}` is `blank` the following results: `tags = ["t1\n", "t2", "t2\n", "t3", "t3\n", "t4"].

To fix this issue, I modified the regexp to use ` `, `"`, `'`, `\n` and `\r\n` as delimiters.

I also tried to use `strip` in the method that handle the annotation lists.